### ISSUE TITLE
Google Chrome on NixOS is dead

### ISSUE THREAD

**Issue by kachick** [kachick/dotfiles#776]
Since https://github.com/kachick/dotfiles/commit/6af2f8002c08be76fe5392f274f73523f4a0b047?

```console
> google-chrome-stable --version
Google Chrome 128.0.6613.113

> google-chrome-stable
[7737:7737:0908/133901.033194:ERROR:object_proxy.cc(576)] Failed to call method: org.freedesktop.ScreenSaver.GetActive: object_path= /org/freedesktop/ScreenSaver: org.freedesktop.DBus.Error.NotSupported: This method is not part of the idle inhibition specification: https://specifications.freedesktop.org/idle-inhibit-spec/latest/
ac: Unknown GPU, using 0 for raster_config
Cannot find target for triple amdgcn-- Unable to find target for this triple (no targets are registered)
[0908/133901.061922:ERROR:process_memory_range.cc(75)] read out of range
[7737:7737:0908/133901.436586:ERROR:gpu_process_host.cc(1007)] GPU process exited unexpectedly: exit_code=139
ac: Unknown GPU, using 0 for raster_config
Cannot find target for triple amdgcn-- Unable to find target for this triple (no targets are registered)
[0908/133901.541239:ERROR:process_memory_range.cc(75)] read out of range
[7737:7737:0908/133901.924005:ERROR:gpu_process_host.cc(1007)] GPU process exited unexpectedly: exit_code=139
ac: Unknown GPU, using 0 for raster_config
Cannot find target for triple amdgcn-- Unable to find target for this triple (no targets are registered)
[0908/133902.025319:ERROR:process_memory_range.cc(75)] read out of range
[7737:7737:0908/133902.397020:ERROR:gpu_process_host.cc(1007)] GPU process exited unexpectedly: exit_code=139
ac: Unknown GPU, using 0 for raster_config
Cannot find target for triple amdgcn-- Unable to find target for this triple (no targets are registered)
[0908/133902.502514:ERROR:process_memory_range.cc(75)] read out of range
[7737:7737:0908/133903.077323:ERROR:gpu_process_host.cc(1007)] GPU process exited unexpectedly: exit_code=139
ac: Unknown GPU, using 0 for raster_config
Cannot find target for triple amdgcn-- Unable to find target for this triple (no targets are registered)
[0908/133903.189622:ERROR:process_memory_range.cc(75)] read out of range
[7737:7737:0908/133903.760426:ERROR:gpu_process_host.cc(1007)] GPU process exited unexpectedly: exit_code=139
ac: Unknown GPU, using 0 for raster_config
Cannot find target for triple amdgcn-- Unable to find target for this triple (no targets are registered)
[0908/133903.873073:ERROR:process_memory_range.cc(75)] read out of range
[7737:7737:0908/133904.419350:ERROR:gpu_process_host.cc(1007)] GPU process exited unexpectedly: exit_code=139
ac: Unknown GPU, using 0 for raster_config
Cannot find target for triple amdgcn-- Unable to find target for this triple (no targets are registered)
[0908/133904.480314:ERROR:process_memory_range.cc(75)] read out of range
[7737:7737:0908/133904.826421:ERROR:gpu_process_host.cc(1007)] GPU process exited unexpectedly: exit_code=139
ac: Unknown GPU, using 0 for raster_config
Cannot find target for triple amdgcn-- Unable to find target for this triple (no targets are registered)
[0908/133904.889466:ERROR:process_memory_range.cc(75)] read out of range
[7737:7737:0908/133905.235000:ERROR:gpu_process_host.cc(1007)] GPU process exited unexpectedly: exit_code=139
ac: Unknown GPU, using 0 for raster_config
Cannot find target for triple amdgcn-- Unable to find target for this triple (no targets are registered)
[0908/133905.302853:ERROR:process_memory_range.cc(75)] read out of range
[7737:7737:0908/133905.654691:ERROR:gpu_process_host.cc(1007)] GPU process exited unexpectedly: exit_code=139
[7737:7737:0908/133905.654723:FATAL:gpu_data_manager_impl_private.cc(436)] GPU process isn't usable. Goodbye.
[0908/133905.664194:ERROR:elf_dynamic_array_reader.h(64)] tag not found
[0908/133905.664587:ERROR:elf_dynamic_array_reader.h(64)] tag not found
[0908/133905.664603:ERROR:process_memory_range.cc(75)] read out of range
zsh: trace trap (core dumped)  google-chrome-stable
```

https://github.com/kachick/dotfiles/blob/49fd11ce3b928f12e8cb18a002d21cffc317998b/nixos/configuration.nix#L226

So PWA spotify also dead

However chromium vscode and signal still run.

**Comment by kachick** [comment]
Using floop might be a candidate to realize PWA in firefox - https://github.com/NixOS/nixpkgs/blob/4f673395c760c97bcdfc23e40e33a176de30787f/pkgs/applications/networking/browsers/floorp/default.nix#L64

**Comment by kachick** [comment]
Might be related...

https://github.com/NixOS/nixpkgs/issues/249152
https://superuser.com/questions/988929/how-to-launch-chrome-with-explicit-hardware-acceleration-setting
https://github.com/signalapp/Signal-Desktop/issues/6855

There is no GPU option?

```console
> google-chrome-stable --disable-gpu
[10866:10866:0908/135358.581915:ERROR:object_proxy.cc(576)] Failed to call method: org.freedesktop.ScreenSaver.GetActive: object_path= /org/freedesktop/ScreenSaver: org.freedesktop.DBus.Error.NotSupported: This method is not part of the idle inhibition specification: https://specifications.freedesktop.org/idle-inhibit-spec/latest/
ac: Unknown GPU, using 0 for raster_config
Cannot find target for triple amdgcn-- Unable to find target for this triple (no targets are registered)
[0908/135358.609766:ERROR:process_memory_range.cc(75)] read out of range
[10866:10866:0908/135359.179556:ERROR:gpu_process_host.cc(1007)] GPU process exited unexpectedly: exit_code=139
ac: Unknown GPU, using 0 for raster_config
Cannot find target for triple amdgcn-- Unable to find target for this triple (no targets are registered)
[0908/135359.292407:ERROR:process_memory_range.cc(75)] read out of range
[10866:10866:0908/135359.848648:ERROR:gpu_process_host.cc(1007)] GPU process exited unexpectedly: exit_code=139
ac: Unknown GPU, using 0 for raster_config
Cannot find target for triple amdgcn-- Unable to find target for this triple (no targets are registered)
[0908/135359.962624:ERROR:process_memory_range.cc(75)] read out of range
[10866:10866:0908/135400.539955:ERROR:gpu_process_host.cc(1007)] GPU process exited unexpectedly: exit_code=139
ac: Unknown GPU, using 0 for raster_config
Cannot find target for triple amdgcn-- Unable to find target for this triple (no targets are registered)
[0908/135400.605337:ERROR:process_memory_range.cc(75)] read out of range
[10866:10866:0908/135400.959065:ERROR:gpu_process_host.cc(1007)] GPU process exited unexpectedly: exit_code=139
ac: Unknown GPU, using 0 for raster_config
Cannot find target for triple amdgcn-- Unable to find target for this triple (no targets are registered)
[0908/135401.023104:ERROR:process_memory_range.cc(75)] read out of range
[10866:10866:0908/135401.384469:ERROR:gpu_process_host.cc(1007)] GPU process exited unexpectedly: exit_code=139
ac: Unknown GPU, using 0 for raster_config
Cannot find target for triple amdgcn-- Unable to find target for this triple (no targets are registered)
[0908/135401.461593:ERROR:process_memory_range.cc(75)] read out of range
[10866:10866:0908/135401.824790:ERROR:gpu_process_host.cc(1007)] GPU process exited unexpectedly: exit_code=139
[10866:10866:0908/135401.824821:FATAL:gpu_data_manager_impl_private.cc(436)] GPU process isn't usable. Goodbye.
[0908/135401.833784:ERROR:elf_dynamic_array_reader.h(64)] tag not found
[0908/135401.834241:ERROR:elf_dynamic_array_reader.h(64)] tag not found
[0908/135401.834258:ERROR:process_memory_range.cc(75)] read out of range
zsh: trace trap (core dumped)  google-chrome-stable --disable-gpu


> google-chrome-stable --help | rg --fixed-strings -e '--'
       --user-data-dir=DIR
       --incognito
       --new-window
       --proxy-server=host:port
                --proxy-server="foopy:99"
                --proxy-server="socks://foobar:1080"
                --proxy-server="socks4://foobar:1080"
                --proxy-server="socks5://foobar:66"
                --proxy-server="https=proxy1:80;http=socks4://baz:1080"
       --no-proxy-server
       --proxy-auto-detect
       --proxy-pac-url=URL
       --password-store=<basic|gnome|kwallet>
       --version
       As a GTK+ app, Google Chrome also obeys GTK+ command-line flags, such as --display.  See the GTK documentation for more:
```

**Comment by kachick** [comment]
Fixed with backed to nixos-24.05 stable channel
And keeping the version. Looks like backporting google chrome, so I don't have motivation to use latest channel

https://github.com/NixOS/nixpkgs/commits/nixos-24.05/pkgs/by-name/go/google-chrome/package.nix

![image](https://github.com/user-attachments/assets/74f589d6-00be-41ea-ab14-de8b37693005)
